tests/vm_test.go

```diff
@@ -528,11 +529,14 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			vm := createRunningVM(virtClient, vmi)
 
 			By("Verifying that the status toggles between ErrImagePull and ImagePullBackOff")
-			const times = 2
-			for i := 0; i < times; i++ {
-				Eventually(ThisVM(vm), 300*time.Second, 1*time.Second).Should(HavePrintableStatus(v1.VirtualMachineStatusErrImagePull))
-				Eventually(ThisVM(vm), 300*time.Second, 1*time.Second).Should(HavePrintableStatus(v1.VirtualMachineStatusImagePullBackOff))
+			expectedStates := []v1.VirtualMachinePrintableStatus{
+				// State transitions a single image pull attempt
+				v1.VirtualMachineStatusErrImagePull,
+				v1.VirtualMachineStatusImagePullBackOff,
 			}
```

Comment:
```suggestion
			expectedStates := []v1.VirtualMachinePrintableStatus{
				// State transitions a single image pull attempt
				v1.VirtualMachineStatusErrImagePull,
				v1.VirtualMachineStatusImagePullBackOff,
				v1.VirtualMachineStatusErrImagePull,
				v1.VirtualMachineStatusImagePullBackOff,
			}
```

Reply:
Don't we need to test the toggle twice?

Reply:
Yeah, that's what the original test case checks. Will do that, and increase the timeout.